config/src/integration-test/java/org/springframework/security/config/annotation/authentication/ldap/LdapBaseSpring.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Ideally, we wouldn't add this class.
By using `SpringTestRule` and `mockMvc` we should have this functionality covered.
You can take a look at the other commits in gh-4939 to see how we keep away from extending a base class.